lab-scott/test/fs-modules-tests.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const expect = require('chai').expect;
+const Buffer = require('buffer').Buffer;
+const fs = require('fs');
+const fsMod = require('../lib/fs-module');
+let expectedArr = [ '5468652053656168', '48656c6c6f2e205a', '476f204d27732061' ];
+fsMod.reads(fsMod.hexArr);
```

Comment:
I like your way of thinking with your use of Line 8. I think you'll enjoy when we get into learning about ``before`` and ``beforeEach`` blocks in Mocha testing.